webpack/middleware/unauthenticated_response.js

```diff
@@ -0,0 +1,16 @@
+import {
+  LOG_OUT
+} from '../actions/types';
+
+const parentFromQuestions = store => next => action => {
+  let {payload} = action;
```

Comment:
I initially was using a default value. Also, I just noticed that the function name is a carryover from the middleware I used as a base. I'll fix it in the next PR to add tests for 401/403 differences.